site/schemas/login.schema.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+import { setZodDefaultError } from "../utils";
+
+export const loginSchema = z.object({
+    email: z.string(setZodDefaultError("Enter an email address")).min(1),
+    password: z
+        .string(setZodDefaultError("Enter a password"))
+        .min(1)
+        .refine((val) => val.length >= 8, {
```

Comment:
I'd probably just get rid of this refine completely and let cognito do the actual checking of the passwords, this could theoretically lead to some weird behaviour if we changed the password length requirement

Reply:
done